frontend/src/pages/CalendarPage/index.js

```diff
@@ -123,6 +123,18 @@ class CalendarPage extends React.Component {
     } = this.state;
     const { classes, history } = this.props;
 
+    // color of events based on status
+    const resources = [
+      {
+        fieldName: 'status',
+        title: 'Status',
+        instances: [
+          { id: 'complete', text: 'Laser Tag', color: 'cadetBlue' },
+          { id: 'pending', text: 'Test Event', color: 'plum' },
+          { id: 'ready', text: 'Test Event', color: 'tomato' },
```

Comment:
Since text is an optional field (https://devexpress.github.io/devextreme-reactive/react/scheduler/docs/reference/resources/#resourceinstance) you don't have to include it here in these JS objects. If you do want to include text, I think doing something like `text: 'Readied Event'` and `'text: Pending Event'` would be a lot better.

Reply:
+1 here - I agree with Thai; probably would be nicer to leave out text, not sure what its purpose is here as of now